macros/count_tokens.sql

```diff
@@ -0,0 +1,7 @@
+{% macro count_tokens(column_name) -%}
```

Comment:
Based on [this doc](https://help.openai.com/en/articles/4936856-what-are-tokens-and-how-to-count-them) and internal discussion, approximate count of tokens is appropriate.